templates/_entries/search/default.twig

```diff
@@ -1,17 +1,114 @@
+{% set searchQuery = craft.app.request.getParam('q') %}
+{% set searchSection = craft.app.request.getParam('section') %}
+{% set searchUpdatedDate = craft.app.request.getParam('updatedDate') %}
+{% set searchWhere = craft.app.request.getParam('searchWhere') %}
+
+{# Date for search select option list #}
+{% set yesturdayDate = "now"|date_modify("-1 day")|date('Y-m-d') %}
+{% set pastWeekDate = "now"|date_modify("-7 day")|date('Y-m-d') %}
+{% set pastMonthDate = "now"|date_modify("-1 month")|date('Y-m-d') %}
+{% set pastYearDate = "now"|date_modify("-1 year")|date('Y-m-d') %}
+
+{# Check if advanced search panel need to be open or closed #}
+{% set openAdvancedSearch = false %}
+{% if searchSection or searchUpdatedDate or searchWhere %}
+    {% set openAdvancedSearch = true %}
+{% endif %}
+
+{# Configuration of the search URL and label that need to be use #}
+{% set searchAction, searchLabel = craft.entries.section('search').one().url() ?? null, 'Search this website'|t %}
+{% if headerSettings.customSearchQueryUrl|length %}
+    {% set searchLabel = 'Search Canada.ca'|t %}
+    {% set searchAction = headerSettings.customSearchQueryUrl %}
+{% endif %}
+
+{# Sections select option #}
+{%  set allSections = craft.query()
+    .select('id, name, handle')
+    .from('sections')
+    .all()
+%}
+
 {% extends '_entries/default' %}
 
 {# IN CONSTRUCTION #}
-
 {% block main %}
     {% include '_pageBuilder/builder' with {
         pageBuilder: entry.pageBuilder
     } %}
 
-    <div class="container">
-        <h1>Search form here...</h1>
+        <div class="container">
+            <form action="{{ searchAction }}" method="get" name="cse-search-box" role="search">
+            <div class="row">
+                <div class="col-md-12">
+                    <BR>
+                        <div class="input-group input-group-lg">
+                            <label for="wb-srch-q" class="wb-inv">{{ searchLabel }}</label>
+                            <input class="wb-srch-q form-control"  list="wb-srch-q-ac" maxlength="200" name="q" type="search" placeholder="{{ searchLabel }}" value="{{searchQuery}}">
+                            <datalist></datalist>
+                            <span class="input-group-btn">
+                                <button class="btn btn-primary btn-small" name="wb-srch-sub" type="submit">
+                                    <span class="glyphicon-search glyphicon"></span>
+                                    <span class="wb-inv">{{ 'search-txt'|t }}</span>
```

Comment:
Is `search-txt` the actual string?

Reply:
Replace search-txt with {{ searchLabel }}.